ql/src/java/org/apache/hadoop/hive/ql/udf/ptf/BasePartitionEvaluator.java

```diff
@@ -218,6 +220,9 @@ public BasePartitionEvaluator(
     this.outputOI = outputOI;
     this.nullsLast = nullsLast;
     this.isCountEvaluator = wrappedEvaluator instanceof GenericUDAFCount.GenericUDAFCountEvaluator;
+    // use a periodic logger which ignores very small partitions
+    this.stopwatch = new PeriodicLoggerWithStopwatch(
```

Comment:
right, I'm also thinking about completely removing this logger from here, and add it in a separate ticket, because it can be also used in FileSinkOperator